applications/StructuralMechanicsApplication/custom_elements/cable_element_3D2N.cpp

```diff
@@ -0,0 +1,174 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:     BSD License
+//           license: structural_mechanics_application/license.txt
+//
+//  Main authors: Klaus B. Sautter
+//                   
+//                   
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_elements/cable_element_3D2N.hpp"
+#include "structural_mechanics_application_variables.h"
+#include "includes/define.h"
+
+
+namespace Kratos
+{
+	CableElement3D2N::CableElement3D2N(IndexType NewId, 
+									GeometryType::Pointer pGeometry)
+									: TrussElement3D2N(NewId, pGeometry)
+	{}
+
+	CableElement3D2N::CableElement3D2N(IndexType NewId,
+									GeometryType::Pointer pGeometry,
+									PropertiesType::Pointer pProperties) 
+									: TrussElement3D2N(NewId, pGeometry, pProperties) 
+	{}
+
+	Element::Pointer CableElement3D2N::Create(IndexType NewId,
+									NodesArrayType const& rThisNodes,
+									PropertiesType::Pointer pProperties) const
+	{
+		const GeometryType& rGeom = this->GetGeometry();
+		return Kratos::make_shared<CableElement3D2N>(
+			NewId, rGeom.Create(rThisNodes),pProperties);
+	}
+
+	CableElement3D2N::~CableElement3D2N(){}
+
+
+
+	bounded_matrix<double,TrussElement3D2N::msLocalSize,
+	TrussElement3D2N::msLocalSize> CableElement3D2N::CreateElementStiffnessMatrix(
+		ProcessInfo& rCurrentProcessInfo){
+
+		KRATOS_TRY
+		bounded_matrix<double,msLocalSize,msLocalSize>
+		 LocalStiffnessMatrix = ZeroMatrix(msLocalSize, msLocalSize);
+
+		if (this->mIsCompressed)
+		{
+			LocalStiffnessMatrix = ZeroMatrix(msLocalSize, msLocalSize);
+		}
+
+		else
+		{
+			this->CalculateElasticStiffnessMatrix(LocalStiffnessMatrix,
+				rCurrentProcessInfo);
+			bounded_matrix<double,msLocalSize,msLocalSize> K_geo = ZeroMatrix(msLocalSize, msLocalSize);
+			this->CalculateGeometricStiffnessMatrix(K_geo, rCurrentProcessInfo);
+
+			LocalStiffnessMatrix += K_geo;
+		}
+
+		return LocalStiffnessMatrix;
+		KRATOS_CATCH("")
+	}
+
+
+	void CableElement3D2N::CalculateLocalSystem(MatrixType& rLeftHandSideMatrix,
+										VectorType& rRightHandSideVector,

```

Comment:
@KlausBSautter I think there are some tabs instead of spaces here too.
Maybe you can automatically remove all tabs and replace them with 4 spaces